packages/suite/src/hooks/wallet/coinmarket/form/common/useCoinmarketComposeTransaction.ts

```diff
@@ -42,7 +43,13 @@ export const useCoinmarketComposeTransaction = <T extends CoinmarketSellExchange
     const coinFees = fees[symbol];
     const levels = getFeeLevels(networkType, coinFees);
     const feeInfo = useMemo(() => ({ ...coinFees, levels }), [coinFees, levels]);
-    const initState = useMemo(() => ({ account, network, feeInfo }), [account, network, feeInfo]);
+    const initState = useMemo(() => {
+        // TODO remove NetworkCompatible type
+        // getNetwork is guaranteed to find a NetworkCompatible, as it is called with symbol from an existing Network
+        const networkCompatible = getNetwork(network.symbol) as NetworkCompatible;
+
+        return { account, network: networkCompatible, feeInfo };
+    }, [account, network.symbol, feeInfo]);
```

Comment:
`useCompose` still requires `networkCompatibility`, and it is used at multiple places. Refactoring it would bloat the scope of this PR a lot, so this is where the new inteface transforms to old.